source/plot/LinePlotData2D.ooc

```diff
@@ -12,30 +12,42 @@ LineStyle: enum {
 
 LinePlotData2D: class extends PlotData2D {
 	lineStyle: LineStyle { get set }
+	dataSeries: VectorList<FloatPoint2D> { get set }
```

Comment:
FloatPoint2DVectorList is more like a temporary speed hack to go around memcpy in generics than something to make permanent use of.